tools/check-style.py

```diff
@@ -0,0 +1,132 @@
+#!/bin/env python3
+
+from shutil import which
+import argparse
+import os
+import subprocess
+import sys
+
+DIFF_INC_COLLOR = "\033[92m"
+DIFF_REM_COLOR = "\033[31m"
+DIFF_REF_COLOR = "\033[36m"
+
+UNCRUSTIFY_VERSION = "0.60"
+
+def run_command(cmd, msg_prefix="ERROR: "):
+    try:
+        output = subprocess.check_output(cmd, stderr=subprocess.STDOUT,
+                                         shell=True, universal_newlines=True)
+        return output
+    except subprocess.CalledProcessError as e:
+        try:
+            print("%s%s" % (msg_prefix, e.output))
+        except BrokenPipeError as e:
+            return None
+        return None
+
+def check_diff():
+    diff = which("diff")
```

Comment:
Hum, why not use difflib directly?
https://docs.python.org/3.4/library/difflib.html